lectures/cobweb.md

```diff
@@ -408,15 +408,15 @@ That is,
 
 ```{math}
 :label: pe_adaptive
-p_{t-1}^e = \alpha p_{t-1} + (1-\alpha) p^e_{t-2}
+p_{t-1}^t = \alpha p_{t-1} + (1-\alpha) p^t_{t-2}
```

Comment:
@HumphreyYang or @mmcky , just glancing at the diffs, I feel like the `t` on the rhs should be `t-1`.

In other words, this is the time $t-2$ expectation of prices at $t-1$.  Similar comments apply to the changes below.  Can you please review.

Reply:
Many thanks @jstac for raising this.

On a second look, I think the notation $p^e_t$ is correct as it represents the expectation of $p$ at time $t$ as I quote in the original lecture:

"Because the crop of soy beans for time $t$ is planted at $t-1$, supply of soy beans at time $t$ depends on *expected* prices at time $t$, which we denote $p^e_{t-1}$."

So the full notation would be $p^e_{t-1} = \mathbb{E}(p_t | H)$, where $H = (p_{t-1}, p_{t-2} \dots)$ is a sequence of relevant history prices.

I think the time script time index here is also correct as it means the expectation of price at time $t-1$ (i.e., the expectation of price for next period standing at $t-1$) is based on a weighted sum of the price at current period $t-1$ and the previous period price $t$ (i.e. $p^e_{t-1} = \mathbb{E}(p_t | p_{t-1}, p_{t-2}) =  \alpha p_{t-1} + (1-\alpha) p_{t-2})$.

I think this notation is used in [John F. Muth's 1961 paper](https://www.jstor.org/stable/1909635):

![Screenshot from 2024-05-08 22-32-09](https://github.com/QuantEcon/lecture-python-intro/assets/39026988/c73cdebb-3d11-4fad-a354-0cf7f5249f0b)

 But I totally agree this notation is confusing!

Reply:
OK @HumphreyYang , maybe you are right about this.

Anyway, it seems I'm outvoted because no one prefers my version :-)

It would be great if someone could switch from $p^e_{t-1}$ to $p^e_t$ throughout.

Reply:
Many thanks @jstac and @mmcky. I will do that.

Reply:
Hi @mmcky, I have made the changes. I feel it is more intuitive compared to the previous notation. Please let me know your thoughts.